packages/openapi-ts/src/compiler/index.ts

```diff
@@ -6,17 +6,29 @@ import * as module from './module';
 import * as types from './types';
 import { tsNodeToString } from './utils';
 
-export class TypeScriptFile extends Array<ts.Node> {
-    public override toString(seperator: string = '\n') {
-        return this.map(v => tsNodeToString(v)).join(seperator);
+export class TypeScriptFile {
+    private _imports: Array<ts.Node> = [];
+    private _items: Array<ts.Node> = [];
+
+    public addImport(...params: Parameters<typeof module.createNamedImportDeclarations>): void {
```

Comment:
Maybe addNamedImport if we have `compiler.import.named`? Implies there might be different types in the future